src/patterns-hmrc-govuk-team/hmrc-email-updates-videos-webinars-community-forums/index.njk

```diff
@@ -0,0 +1,310 @@
+---
+title: HMRC email updates, videos, webinars and community forums
+menuText: HMRC email updates, videos, webinars and community forums
+layout: twoColumnPage.njk
+---
+{% from "_example.njk" import example %}
+{% from "_codeSnippet.njk" import codeSnippet %}
+
+<h2 class="govuk-heading-l">When to use</h2>
+
+<p class="govuk-body">Use this pattern to design new or existing content that includes email updates, videos, webinars and community forums. It may also sometimes include e-learning and guidance.</p>
+
+<h2 class="govuk-heading-l">When not to use</h2>
+
+<p class="govuk-body">Do not use this pattern if you’re just <a href="https://www.gov.uk/guidance/how-to-publish-on-gov-uk/markdown#video"> adding a video to a piece of guidance</a>.</p>
```

Comment:
The link should have a class of `govuk-link`

Reply:
Also, I'd remove the space before the word `adding`. I don't _think_ it'll make a difference as HTML would only see add a single space but it's more semantically correct and _might_ have an effect on how it reads out to screenreaders

Reply:
Fixed.